src/go/k8s/main.go

```diff
@@ -273,12 +276,12 @@ func main() {
 
 	//+kubebuilder:scaffold:builder
 
-	if err := mgr.AddHealthzCheck("health", healthz.Ping); err != nil {
+	if err := mgr.AddHealthzCheck("healthz", healthz.Ping); err != nil {
 		setupLog.Error(err, "Unable to set up health check")
 		os.Exit(1)
 	}
 
-	if err := mgr.AddReadyzCheck("check", healthz.Ping); err != nil {
```

Comment:
@joejulian the check path was never "checked" in the livenless checks. Am I missing something here?

Reply:
Great find 👍 

When I scaffold this project v3 was using `"check"`.

REF:
https://github.com/redpanda-data/redpanda/commit/baea03d5b690cf4ac94cb607760c82e785bb13e9
https://github.com/kubernetes-sigs/kubebuilder/pull/1910